packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -219,6 +219,7 @@ export class Field extends PureComponent {
                   { ...inputEvents }
                   // shipping options have checked attr assigned so prioritize its value
                   checked={ checked || isChecked }
+                  defaultChecked={ defaultChecked }
```

Comment:
the defaultChecked prop is used for uncontrolled components.
You shouldn't mix checked and defaultChecked props.
Please improve the logic that defines the checked state instead.